scripts/circleci-jobs/openshift3-integration-tests.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+import os
```

Comment:
what is specific here to openshift3? wondering about the file name and why it is used only in this flow

Reply:
Nothing specific, the goal is to reduce our CCI script size. Starting with this PR by not adding more to the existing debt

Reply:
so maybe it makes sense to call it in a more general name that hints about the file purpose? anyway, your call